packages/flutter/test/material/tabs_test.dart

```diff
@@ -2350,4 +2350,76 @@ void main() {
     expect(find.text('Tab1'), findsOneWidget);
     expect(find.text('Tab2'), findsOneWidget);
   });
+
+  testWidgets('DefaultTabController should allow for a length of zero', (WidgetTester tester) async {
+    List<String> tabTextContent = <String>[];
```

Comment:
You could say that this is a regression test for https://github.com/flutter/flutter/issues/20292